src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java

```diff
@@ -373,7 +421,7 @@ public void scan(HttpMessage msg, String param, String value) {
                     || errorMatcher.find()) {
 
                 if (log.isDebugEnabled()) {
-                    log.debug("It IS possible to check for local file Path Traversal on the url filename on ["
+                    log.debug("It is possible to check for local file Path Traversal on the url filename on ["

```

Comment:
Regarding formatting I cannot revert it, but I've to do this because someone is using tabs instead of spaces and classes every time become not easily readeable... I can try to revert everything, then pull a request for formatting that need to be merged and then copy again this code an pull it again, but can we avoid this this time because the change are related to a plugin managed by me ad Colm?